src/backend/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.7"
+
+services:
+  transaction-api:
+    build: ./services/transaction-api
+    container_name: transactions
+    restart: always
+    volumes:
+      - ./services/transaction-api:/src
+    environment:
+      - APP_NAME=TransactionService
+    expose:
+      - 5000
+
+  api-gateway:
+    build: ./services/api-gateway
+    container_name: api-gateway
+    restart: always
+    ports:
+      - "80:80"
+
+  blockchain:
+    build: ./blockchain
+    container_name: blockchain
+    restart: always
+    volumes:
+      - ./blockchain:/src
+    environment:
+      - APP_NAME=Blockchain
+    expose:
+      - 5000
```

Comment:
Docker-compose was updated to work with the shared folder in a previous PR created by Akshay. These changes should go in that, under backend -> services -> docker-compose.yml

Reply:
Make sure you follow the current structure of the docker-compose files for blockchain as well